x/vesting/keeper/gov.go

```diff
@@ -33,3 +33,35 @@ func (k Keeper) DeleteGovClawbackDisabled(ctx sdk.Context, addr sdk.AccAddress)
 	key := append(types.KeyPrefixGovClawbackDisabledKey, addr.Bytes()...)
 	ctx.KVStore(k.storeKey).Delete(key)
 }
+
+// HasActiveClawbackProposal checks if there is an active clawback proposal for the given
+// vesting account address.
+func (k Keeper) HasActiveClawbackProposal(ctx sdk.Context, addr sdk.AccAddress) bool {
+	key := buildActiveAccountClawbackProposalKey(addr)
+
+	return ctx.KVStore(k.storeKey).Has(key)
+}
+
+// SetActiveClawbackProposal sets the given vesting account address as subject to an active governance clawback
+// proposal by writing it to store under the corresponding key.
+func (k Keeper) SetActiveClawbackProposal(ctx sdk.Context, addr sdk.AccAddress) {
+	key := buildActiveAccountClawbackProposalKey(addr)
+	ctx.KVStore(k.storeKey).Set(key, []byte{0x01})
```

Comment:
Can't we just save empty bytes associated to this key?

Reply:
yes we could, I used this to be in line with the way we store the key for the governance clawback. At any point in there will only be at max 1 or two active proposals though so this would decide about one or two more bytes stored on chain so I think we can just leave it as is 🤷